include/vctr/Containers/Array.h

```diff
@@ -39,7 +39,8 @@ struct StorageExtent<T, e>
 };
 } // namespace detail
 
-/** The stack-based container type.
+/** @ingroup Core
+    @brief The stack-based container type.
```

Comment:
You don't need any `@brief` commands if you enable `JAVADOC_AUTOBRIEF`. I suggest doing so.

Reply:
Good point, changed that